.github/workflows/cd.yml

```
@@ -0,0 +1,168 @@
+        fi
+
+    - name: Cleanup old deployment
+      if: success()
+      run: |
+        OLD_COLOR=${{ steps.color.outputs.color == 'blue' && 'green' || 'blue' }}
+        helm uninstall astraguard-$OLD_COLOR --namespace astraguard || true
+
+    - name: Rollback on failure
+      if: failure()
+      run: |
+        echo "Deployment failed, rolling back..."
```

actionlint (1.7.9)

unexpected EOF while lexing expression

(expression)